.contrib/Parser/DATAS/02 - Outdoor Zones/01 Kalimdor/Dustwallow Marsh.lua

```diff
@@ -657,6 +681,9 @@ root(ROOTS.Zones, m(KALIMDOR, {
 					["coord"] = { 53.5, 56.9, DUSTWALLOW_MARSH },
 					["timeline"] = { ADDED_4_0_3 },
 					["groups"] = {
+						objective(1, {	-- 0/10 Oozes Dissolved
```

Comment:
This also required using a quest item from your bags, killing them without using the item yielded no quest completion.